apps/react-native-webview-example/app/room/index.tsx

```diff
@@ -21,12 +20,15 @@ export default function Room() {
                     <Text key={index}>{entry}</Text>
                 ))}
             </ScrollView>
+            {/* <Button onPress={() => wherebyRoomRef.current?.knock()} title="Knock" /> */}
             <View style={{ flex: 1, height: "100%" }}>
                 <WherebyEmbed
                     ref={wherebyRoomRef}
                     style={styles.container}
-                    roomUrl={ROOM_URL}
-                    queryParams={queryParams}
+                    room={ROOM_URL}
+                    minimal={"on"}
+                    skipMediaPermissionPrompt={"on"}
+                    displayName="React Native Client"
```

Comment:
Is this a placeholder display name ? Or something for us to track?

Reply:
Just a placeholder. Removed it for now. We can add more example props in this app later if needed